README.rst

```diff
@@ -43,13 +33,43 @@ To deploy it, I'm using gunicorn and supervisord.
 
 Don't forget to set the right permission for your files !
 
-Also, create a `settings.py` file with the appropriate values if you need to
-use a different database for instance. You can also set `APPLICATION_ROOT` if
-you want to prefix your URLs to serve ihatemonney in the *folder* of a domain,
-e.g:
+Configure it
+============
+
+In a production environment
+---------------------------
+
+Make a copy of budget/default_settings.py and name it ihatemoney.cfg.
```

Comment:
nitpick: use double backquotes here to make file locations appear inside `pre` html tags. This applies to all file locations.